packages/devtron-fe-lib/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@devtron-labs/devtron-fe-lib",
+  "version": "1.0.0",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "scripts": {},
+  "author": "devtron-labs",
```

Comment:
```suggestion
```

Reply:
Added license Apache-2.0 to all the package.json